tests/suites/test_suite_psa_crypto_se_driver_hal_mocks.function

```diff
@@ -125,12 +114,16 @@ static psa_status_t mock_import( psa_key_slot_number_t slot_number,
     (void) p_data;
 
     mock_import_data.called++;
-    mock_import_data.slot_number = slot_number;
-    mock_import_data.lifetime = lifetime;
-    mock_import_data.type = type;
-    mock_import_data.algorithm = algorithm;
-    mock_import_data.usage = usage;
-    mock_import_data.data_length = data_length;
+
+    if( mock_import_data.return_value == PSA_SUCCESS )
```

Comment:
This should be set regardlesss of the result.

Reply:
Do we want to set all these values even on failure? The test seems to indicate that `type` at least should remain unset if `mock_import_return_value` isn't `PSA_SUCCESS`

Reply:
The mock structures only indicate with what arguments was the function called, regardless of any error.